test.js

```diff
@@ -946,7 +916,7 @@ test('createAsyncIterator creates Iterator for Array', () => {
 })
 
 test('createAsyncIterator creates Iterator for Iterator', () => {
-  var myIterator = getIterator(['Alpha', 'Bravo', 'Charlie'])
+  var myIterator = getIterator((['Alpha', 'Bravo', 'Charlie']: Iterable<string>))
```

Comment:
Interesting Note:
In the Flow Core type defs, there is this code:
```typescript
interface $Iterable<+Yield, +Return, -Next> {...}
type Iterable<+V> = $Iterable<V, void, void>;
```

If you replace your usage of `Iterable<V>` with `$Iterable<V, void, void>` Flow is slightly better and choosing the correct overloads.